lib/run/options.js

```diff
@@ -57,35 +57,41 @@ var _ = require('lodash'),
      */
     configLoaders = {
         collection: function (value, callback) {
-            var done = function (err, collection) {
-                if (err) {
-                    return callback(err);
-                }
+            var V2_COLLECTION_PROPERTY = 'info.schema',
```

Comment:
@kunagpal The transformer now has functions called `isv1` and `isv2` to help out with this.